omnigibson/objects/stateful_object.py

```diff
@@ -471,6 +471,9 @@ def _load_state(self, state):
                 else:
                     logging.warning("Missing object state [{}] in the state dump".format(state_name))
 
+        # Clear cache after loading state
+        self.clear_states_cache()
```

Comment:
good catch -- in this case, would you remind removing the redundant line from `_load_state` in scene_base.py?

Reply:
nvm, you already did!